python/whylogs/api/writer/whylabs.py

```diff
@@ -530,7 +530,7 @@ def _write_segmented_reference_result_set(self, file: SegmentedResultSet, **kwar
         upload_statuses = list()
         for view, url in zip(files, upload_urls):
             with tempfile.NamedTemporaryFile() as tmp_file:
-                if kwargs.get("use_v0") is None or kwargs.get("use_v0"):
+                if kwargs.get("use_v0") is not None or kwargs.get("use_v0"):
```

Comment:
I think the original code might be correct if `use_v0 = True` is meant to be the default. With the proposed change, calling `foo._write_segmented_reference_result_set(file, use_v0=False)` would act the same as calling `foo._write_segmented_reference_result_set(file, use_v0=True)` because `kwargs.get("use_v0") is not None` would be `True` in either case.

Reply:
correct, but we might be able to remove this special handling for v0 format if we don't need this anymore?